tests/tests/test_testcase.py

```diff
@@ -55,14 +55,14 @@ def test_fake_time_in_future(self):
         self.fakeTimeInFuture(1, "months")
         self.assertEqual(pendulum.now().diff(real_now).in_months(), 1)
 
-    def test_fake_time_in_past(self):
```

Comment:
Did you had issues with this test 🤔 ??? It's working on master normally

Reply:
I had to remove some of these tests as they were throwing a weird SQLite database connection already closed issue. Which may be a bigger issue with the ORM I'm not sure. I dove into it but I didn't have enough time to complete the rabbit hole journey as my debugging I couldn't find the issue

Reply:
Okay let's dig into that later !